extensions/git-branch-name-generator/package.json

```
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "git-branch-name-generator",
+  "title": "Git Branch Name Generator",
+  "description": "Convert a string into a branch name",
+  "icon": "command-icon.png",
+  "author": "mbonner",
+  "categories": [
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Generate Git Branch Name",
+      "subtitle": "From type and string inputs",
```

```suggestion
      "subtitle": "",
```
Doesn't add any contextual value